python/ray/experimental/sgd/pytorch/distributed_pytorch_runner.py

```diff
@@ -109,26 +116,15 @@ def get_state(self):
             for k, v in state_dict.items():
                 state_dict[k] = v.cpu()
             cpu_state_dicts += [state_dict]
-        return {
-            "epoch": self.epoch,
-            "models": cpu_state_dicts,
-            "optimizers": [opt.state_dict() for opt in self.optimizers],
-            "stats": self.stats()
-        }
-
-    def set_state(self, state):
-        """Sets the state of the model."""
-        # TODO: restore timer stats
-        for model, model_state_dict in zip(self.models, state["models"]):
+        return cpu_state_dicts
+
+    def _set_model_state_dicts(self, model_state_dicts):
+        for model, model_state_dict in zip(self.models, model_state_dicts):
             model.module.load_state_dict(model_state_dict)
-        for optimizer, opt_state_dict in zip(self.optimizers,
-                                             state["optimizers"]):
-            optimizer.load_state_dict(opt_state_dict)
-        self.epoch = state["stats"]["epoch"]
 
-    def shutdown(self):
+    # def shutdown(self):
```

Comment:
Unsure if this is worth deprecating/warning instead of removing?

Reply:
Ah, on master, it just calls the super class already, so removing it doesn't change behavior.